.travis.yml

```diff
@@ -34,6 +34,12 @@ matrix:
       env: "TEST_SUITE=LINT-PHP"
     - php: 7.4snapshot
       env: "TEST_SUITE=TEST-JS"
+    - php: 7.4snapshot
```

Comment:
7.4 is still not available?

Reply:
It is, but using it caused some weird issues with composer